edalize/edatool.py

```diff
@@ -268,6 +271,10 @@ def __init__(self, name, file_type, logical_name):
         for f in self.files:
             if 'is_include_file' in f and f['is_include_file']:
                 _incdir = os.path.dirname(f['name']) or '.'
+                if 'include_path' in f and f['include_path']:
+                    if os.path.isabs(f['include_path']):
+                        raise EdamException("'include_path' of file {} cannot be absolute".format(f))
```

Comment:
I think we should allow absolute paths for now. There might be cases where users need to include some system library. It's not portable, but at least we wouldn't prevent the use from doing it. Maybe we could print a warning instead